modules/working-file-repository-service-impl/src/main/java/org/opencastproject/workingfilerepository/impl/WorkingFileRepositoryImpl.java

```diff
@@ -141,6 +152,47 @@ public void activate(ComponentContext cc) throws IOException {
 
     registeredMXBean = JmxUtil.registerMXBean(workingFileRepositoryBean, JMX_WORKING_FILE_REPOSITORY_TYPE);
 
+    // Determine garbage collection period
+    int garbageCollectionPeriodInSeconds = -1;
+    String period = StringUtils.trimToNull(
+            cc.getBundleContext().getProperty(WORKING_FILE_REPOSITORY_CLEANUP_PERIOD_KEY));
+    if (period != null) {
+      try {
+        garbageCollectionPeriodInSeconds = Integer.parseInt(period);
```

Comment:
I'm a fan of failing fast if there is a known problem instead of unexpectedly going into a somewhat unknown state. In other words, if the configuration is completely broken, let the service fail and let the user fix the configuration.

Reply:
@lkiesow  Tried to fix this, now the build fails in another module for org.opencastproject.terminationstate.aws.AutoScalingTerminationStateServiceTest 
can you check this?

Reply:
That seems to be https://github.com/opencast/opencast/issues/1111